### ISSUE TITLE
Gutenboarding: General lack of :focus styles and keyboard navigation

### ISSUE THREAD

**Issue by shaunandrews** [Automattic/wp-calypso#42276]
When trying to use `tab` to navigate around the screen, I'm unable to navigate to the top bar without a mouse. Also, as I use `tab` to navigate, there's a general lack of `:focus` styles which make it hard to know where I am on the page. Here's a GIF that shows me trying to navigate the screen without a pointer device:

![tab control](https://user-images.githubusercontent.com/191598/82059473-14696880-9694-11ea-9d02-3ea08526b0e6.gif)

**Comment by shaunandrews** [comment]
The GIF in my original report is no longer relevant, but a quick run through the existing signup flow shows that its still impossible to navigate through with a keyboard.
